src/navigation.md

```diff
@@ -142,6 +142,7 @@ search:
         - [Delaunay triangulation and Voronoi diagram](geometry/delaunay.md)
         - [Vertical decomposition](geometry/vertical_decomposition.md)
         - [Half-plane intersection - S&I Algorithm in O(N log N)](geometry/halfplane-intersection.md)
+        - [Manhattan Distance](geometry/manhattan-distance.md)
```

Comment:
Please also add it to the list of new articles in README.md.

Reply:
Done